docs/load-test-how-to-use.md

```diff
@@ -0,0 +1,115 @@
+
+#### Analysing load-test results
+
+"My load tests ran successfully, what to make of it?"
+    - In case of unbounded load-tests, when they finish, `go run ./cmd/ltctl loadtest status` would give you a count of maximum concurrent users which is a metric to compare the performance of that version of mattermost-server.
+    - In case of two bounded loadtests with same `MaxConcurrentUsers` count, one can [generate a report](https://github.com/mattermost/mattermost-load-test-ng/blob/master/docs/compare.md) comparing performance of various server metrics. 
+    - In case of both bounded and unbounded loadtests, one can create Grafana dashboards to analyze the performance of the new features by filtering API metrics to only what's relevant to the new handler. Here's an example.
+        ![sample-dashboard-creation](https://i.imgur.com/zzRfh8b.png)
+
+
+#### Testing changes in cloud without terraform
+Even without the framework, a general load-test workflow in the cloud will be similar to the following:
+
+ - Create a database (to be used by Mattermost servers).
+ - Deploy Mattermost servers - let's call them app servers for convenience.
+ - Create 'agent' machines to ping app servers using [controllers](https://github.com/mattermost/mattermost-load-test-ng/tree/61c44f35224b76d3098199b0cd2b67db2222b549/loadtest/control).
+ - Populate a database, then start app servers and agents (i.e., loadtest).
+ - Collect metrics from app servers and agent deployments to either manage an unbounded load test or analyze API performance after the load-test completes.
+
+
+#### Creating a PR
+
+After all the code changes:
+ - [Add tests](https://github.com/mattermost/mattermost-load-test-ng/blob/master/docs/coverage.md#testing) if required. 
+ - Run `make check-style`.
+ - `go get -u github.com/mattermost/mattermost-server/v6@<commit-hash-in-master>` && `go mod tidy`
+ - Create the PR.
+
+#### Note:
+
+ - **For populating the database manually** :
+    - [InstanceConfiguration](https://github.com/mattermost/mattermost-load-test-ng/blob/master/docs/loadtest_config.md#instanceconfiguration) section would be as minimal as possible to reduce `db init` time.
+    - Post a message in the [Developers:Performance](https://community.mattermost.com/core/channels/developers-performance) channel to request a migration file.
+    - If you're using [comparision](https://github.com/mattermost/mattermost-load-test-ng/blob/master/docs/comparison.md), there is [an option to load db from a backup](https://github.com/mattermost/mattermost-load-test-ng/blob/master/docs/comparison_config.md#dbdumpurl). If not,
+        - `ssh` into the app machine, and `psql` into the connected database.
```

Comment:
We can specify here that the DB endpoint can be obtained with `ltctl deployment info`.